README.md

```diff
@@ -286,6 +171,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Hello, kindly have a look at this link, there is a spelling mismatch and as a result, I get the below error. Kindly correct spelling on either file name or link name as you please and make sure both file name and relative link path matches

![Screenshot from 2023-10-12 09-04-44](https://github.com/CobbyElsonfx/vet_clinic/assets/112261237/36932a8f-3289-4638-ba4f-a307ef057005)

Reply:
Thanks ,I will resolve the problem